modelservice/src/main/java/com/expedia/adaptivealerting/modelservice/repo/ModelRepository.java

```diff
@@ -49,7 +49,7 @@
     @RestResource(rel = "findByDetectorId", path = "findByDetectorId")
     List<Model> findByDetectorIdOrderByBuildTimestampDesc(@Param("detectorId") Long detectorId);
     
-    @Query(nativeQuery = true, value = "SELECT * FROM model where last_build_ts =(SELECT MAX(m1.last_build_ts) from model m1 , detector d1 where d1.id=m1.detector_id and d1.uuid=:uuid) LIMIT 1")
+    @Query(nativeQuery = true, value = "SELECT * FROM model m where m.id =(SELECT m1.id from model m1 , detector d1 where d1.id=m1.detector_id and d1.uuid=:uuid) LIMIT 1")
```

Comment:
Do we need the subselect here? Can't we just do

`SELECT m.* from model m, detector d where d.id=m.detector_id and d.uuid=:uuid LIMIT 1`